index.html

```diff
@@ -11,169 +11,223 @@
+                          <ul class="flex-item navlang-flex">
+                              <li><a href="#" class="subflex-item artag">CSS</a></li>
+                              <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                              <li><a href="#" class="subflex-item artag">html</a></li>
+                              <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                              <li><a href="#" class="subflex-item artag">bootstrap</a></li>
+                              <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                              <li><a href="#" class="subflex-item artag">Ruby</a></li>
+                              </ul>
+                          <ul class="seep"><li><a href="#" class="subflex-item seeproject">See Project</a></li></ul>
+                      </div>
+                      
+                  </section>
 
+              </div>
+  
+          </article>
+          <article class="contact-section">
+              <section class="cc-sect cc-top">
+                  <p class="contact mble">CONTACT US</p>
+                  <p class="contact dstop">Let's chat!</p>
+                  <p class="text-invite">
+                      If you have an application you are interested in developing, 
+                      a feature that you need built or a project that needs coding. 
+                       love to help with it 
+                  </p>
+              </section>
+              <section class="cc-sect">
+                  <form action="https://formspree.io/f/meqdogdz" method="post" class="myform" id="someform">
+                      <input type="text" maxlength="30" name="Name" required placeholder="Full Name">
+                      <input type="email" name="Email" required placeholder="Email" id="email">
+                      <textarea maxlength="500" name="message" required placeholder="Enter You Message Here"></textarea>
+                      <button type="submit">See Project</button>
+                
+                    </form>
+                    <img src="image/contact-bar.png" alt="horizontal bar" class="contact-image">
+              </section>
+  
+          </article>
+  
+      </main>
+      <footer class="foota">
+          <nav class="social">
+              <ul>
+                <li><a href="#"><img src="image/github.png" alt="github"></a></li>
+                <li><a href="#"><img src="image/in.png" alt="linkIn"></a></li>
+                <li><a href="#"><img src="image/twofingers.png" alt="Two Fingers"></a></li>
+                <li><a href="#"><img src="image/Twitter.png" alt="Twitter"></a></li>
+                <li class="m"><a href="#"><img src="image/M.png" alt="M"></a></li>   
+              </ul>
+              <ul><li><a href="#">yesomarib1@gmail.com </a></li></ul>
+            </nav>
 
+      </footer>
+      
```

Comment:
Kindly remove the extra space from the bottom of the page to match the Figma design in both Mobile and Desktop versions

Your design:

![image](https://user-images.githubusercontent.com/73714615/187038809-7e33a960-2b0d-41ed-a235-e375141ddb3b.png)
![image](https://user-images.githubusercontent.com/73714615/187038818-f7dc2b78-11f3-41f4-8499-3a81ee829ca9.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187038841-d6bd94d1-8dec-4ab3-9086-fb3cc7c59671.png)
![image](https://user-images.githubusercontent.com/73714615/187038849-1f7a5916-f130-41e1-bf37-3486c6e96f51.png)